gatr/layers/linear.py

```diff
@@ -9,6 +9,8 @@
 from gatr.interface import embed_scalar
 from gatr.primitives.linear import NUM_PIN_LINEAR_BASIS_ELEMENTS, equi_linear
 
+from experiments.logger import LOGGER
```

Comment:
I think there is no need for doing this. You can just reset this to the main branch (and same for gatr.py below)